### PR TITLE
Changes tweet_id to tweet_ids. Updated website card properties. Adds …

### PR DESCRIPTION
**Pull request by GonzoGhanem**
…support for array like responses. Makes campaign ID writable so we can update them

Pluralized the tweet_ids property

Sets the properties for website card

Adds fix for response data comming as an array, this should be fixed server side anyways

Fixes broken tests

Making campaign ID writable so we can update campaigns

### CODE REVIEW

**Comment by GonzoGhanem** [comment]
@brandonblack We've this forked version of the gem and we manage to fix some things in order to satisfy our needs. Please take a look and let me know what do you think. Thanks!

File: lib/twitter-ads/campaign.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ class Campaign
 
     attr_reader :account
 
-    property :id, read_only: true
+    property :id
```
Comment (brandonblack) [inline lib/twitter-ads/campaign.rb]: Allowing `:id` to be writeable will interfere with `save()` logic. You probably don't want to do this even in your own custom version. Rather than manually setting the `:id`, you should just load the object by `:id` from the API.
```ruby
# how to load a specific campaign
campaign_id = '3to3us'
campaign = account.campaigns(campaign_id)

# alternative and equivalent syntax
campaign_id = '3to3us'
campaign = TwitterAds::Campaign.load(account, campaign_id)
```

File: lib/twitter-ads/creative/promoted_tweet.rb [hunk 2]
```diff
@@ -19,7 +19,7 @@ class PromotedTweet
       property :deleted, type: :bool, read_only: true
 
       property :line_item_id
-      property :tweet_id
```
Comment (brandonblack) [inline lib/twitter-ads/creative/promoted_tweet.rb]: Make sure you're running at least v0.2.4 where we've already fixed this issue.

File: spec/twitter-ads/creative/promoted_tweet_spec.rb [hunk 3]
```diff
@@ -23,7 +23,7 @@
   # check model properties
   subject { described_class.new(account) }
   read  = %w(id approval_status created_at updated_at deleted)
-  write = %w(line_item_id tweet_id paused)
```
Comment (brandonblack) [inline spec/twitter-ads/creative/promoted_tweet_spec.rb]: Make sure you're running at least v0.2.4 where we've already fixed this issue.
https://github.com/twitterdev/twitter-ruby-ads-sdk/releases/tag/v0.2.4

File: lib/twitter-ads/resources/dsl.rb [hunk 4]
```diff
@@ -20,6 +20,7 @@ module InstanceMethods
       #
       # @since 0.1.0
       def from_response(object)
+        object = object[0] if object.is_a? Array
```
Comment (brandonblack) [inline lib/twitter-ads/resources/dsl.rb]: Be careful about this change, some API objects do deserialize to an array and this will cause problems. I wouldn't advise making this change unless you're using a modified version of the SDK for limited purposes and this suits that purpose. If you made this change to address the issue with `tweet_ids` in `PromotedTweet` you can revert this change and just update to v0.2.4.

File: lib/twitter-ads/campaign.rb (hunk 1 above)
Comment (GonzoGhanem) [inline lib/twitter-ads/campaign.rb]: Noted! Looks like we did have this properly implemented and never saw it before, thanks for the hint!!

**Comment by GonzoGhanem** [comment]
@brandonblack I will revert all our changes since you have fix them all in v0.2.4 . The only outstanding issue are the website_card's properties which are not correct. But I can update our fork to the lastest and apply my change on top to have working until it's fixed. Do you need me to create an issue for that one ?

File: lib/twitter-ads/creative/website_card.rb [hunk 5]
```diff
@@ -19,17 +19,9 @@ class WebsiteCard
...
-      property :privacy_policy_url
-      property :title
-      property :submit_url
-      property :submit_method
-      property :custom_destination_url
-      property :custom_destination_text
-      property :custom_key_screen_name
-      property :custom_key_name
-      property :custom_key_email
+      property :website_cta
```
Comment (brandonblack) [inline lib/twitter-ads/creative/website_card.rb]: We'll get this sorted out in v0.2.5. Thanks!

**Comment by brandonblack** [comment]
Thanks very much for getting involved! I'm going to close this PR, but will include your website card changes in v0.2.5. If you spot anything else or have any other suggestions please let us know.
